src/banner.html

```diff
@@ -1,11 +1,11 @@
       <ul className="banner-background">
-		<li><a href={`https://www.${window.BASE_URL}/about/smart-city-projects`}><img src="assets/projects.png"/></a></li>
+		<li><a href={`https://www.${window.DISC_UI_URL}/about/smart-city-projects`}><img src="assets/projects.png"/></a></li>
```

Comment:
while we're here, can we remove `www` and also `https://` from here and line 10 in this same file? Those things may not be the case for some deployments.